spec/symengine_spec.rb

```diff
@@ -18,5 +18,12 @@
     it { is_expected.to be_a SymEngine::Rational }
     its(:to_s) { is_expected.to eq '1/3' }
   end
+  
+  describe 'parse' do
+    subject { SymEngine::parse('123 + 321') }
+    
+    it { is_expected.to be_a SymEngine::Integer }
+    it { is_expected.to eq 444 }
```

Comment:
Got it. I just always became suspicious when I see tests for success and not tests for fail :)
In fact, I always prefer to add empty contexts for such a deferred tests pathes, like this:
```ruby
context "When there is parsing error" do
  # TODO: will be covered on weeks 9-10
end
```
Though, I'm not insisting, currently.